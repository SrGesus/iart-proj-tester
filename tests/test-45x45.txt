VB	BC	LV	VC	VD	VD	VE	LV	VB	VD	VC	VE	LV	LH	BE	FE	VE	LH	LV	LH	LV	LV	LV	LH	BD	LV	LH	LH	LV	FE	VD	BB	LH	LV	LH	LV	VC	FD	LH	BE	VE	VE	LV	BC	VB
FE	LH	FB	BB	BD	FB	LH	FE	VB	VD	VD	VD	VB	VC	LV	VC	BE	LH	VB	FD	VD	LH	LH	LV	BE	VD	LH	LH	LH	LV	VC	VC	LH	LV	VC	VE	VB	VD	VD	FE	BE	VB	FC	LV	LH
VB	VD	VE	VD	FD	VD	VC	VD	BD	LH	VD	VB	VC	FB	BB	VD	VB	VC	VD	VB	LH	VE	FE	VB	VB	VC	LH	VC	VD	LV	BD	LV	FD	VE	VB	VD	LV	VB	VE	VD	VD	VD	LV	LH	LH
BB	VE	LV	VD	LV	VC	VE	VB	LH	FB	BC	VE	VB	VE	VD	FC	LV	LH	VB	LV	VE	BD	LH	VC	VB	LH	VE	VE	VE	VC	VD	VC	LH	VB	VB	BC	LH	FE	VC	VB	FC	VC	BC	LH	LH
FD	LV	LV	VD	VC	VC	BC	LV	LV	VD	LV	VE	LV	VE	LH	VD	LH	LH	LV	VB	FD	LV	VB	FC	VC	VD	LH	VE	VC	VB	VD	LV	VE	LV	VE	VB	BC	VC	LV	VB	BB	LV	VE	LV	LH
VC	VE	LV	LH	FB	LV	LV	VC	BE	LV	VE	VE	BB	VC	FE	BB	VB	VB	VC	BC	LH	VB	VB	LV	VB	LH	LH	LH	BE	FC	LH	LV	FD	VD	LH	VB	FC	LV	LH	FC	VE	LV	LH	VE	LH
LV	VE	VB	VD	LH	VC	LH	FC	LV	LH	VC	VD	FD	VE	LH	LH	VC	VB	VC	LV	FB	BE	BC	LH	VC	LV	LH	LV	VE	LV	VD	LH	VE	BE	FB	LH	VD	VD	VE	VB	LV	FD	VC	VC	VB
BE	VE	VE	BD	LV	VE	LV	LV	LH	LV	FB	VE	VD	VC	LV	LH	VB	FE	LV	VC	VC	FE	LH	VC	LH	VB	VB	BE	BB	FD	VD	VE	LH	VB	LV	VC	VB	BB	VE	LH	VB	VB	VD	BD	FD
VD	VC	FE	LH	FC	BC	LH	LH	LV	LV	VC	VE	LH	LH	VD	BE	LH	BC	VE	VB	BE	LV	VC	VB	VB	FD	VB	VB	FB	VD	BE	VD	VC	VB	LV	LV	VD	FD	VD	BD	VD	VE	LV	BE	VE
FB	VC	VE	BE	VE	VE	VC	VC	VD	VC	VD	LH	LV	LH	LV	VD	VE	FC	VE	VB	VE	LH	BE	FE	VB	BC	LH	VB	VB	LV	LV	VE	VC	VC	LV	FE	VE	LH	VC	VE	VB	VD	VB	FB	LV
BB	VB	LH	LV	VE	VE	VD	VE	VD	VC	LV	LH	LV	LV	VB	VB	VB	LV	BC	FE	LH	FD	VC	LH	VB	LV	VE	VE	LH	LH	VB	FB	LV	LV	VC	BC	LV	FB	LV	VC	VB	LV	LV	VC	BE
LV	VD	VC	VC	FD	LH	LV	VC	VD	VC	VC	LH	VD	VC	VD	VD	VD	LH	LH	LH	BD	BE	LH	VB	VD	VD	VE	VD	LV	VB	LH	VB	LV	VC	BC	VC	LV	VC	LH	FE	VB	VC	VB	VE	LV
LH	FC	LV	BC	LV	BB	VB	LV	VE	FB	LH	BD	LH	LV	BE	VD	LH	VE	VB	VC	LH	VD	VD	VE	VE	LH	VD	VD	BD	VB	VD	BB	VB	VB	FE	VD	LH	BC	LH	VC	VD	VB	LH	VD	LH
VB	LH	LV	VD	FE	VC	VB	FB	VC	LV	LV	LV	LH	VD	FC	LH	VB	VE	LV	LV	FE	BE	VC	VC	FE	VC	LV	VD	FE	VC	LH	VB	FE	VE	LV	BE	FB	VC	LV	VD	VD	LH	FC	VC	VB
FC	VD	LV	VC	BB	LH	VB	VC	LH	LH	BB	BE	FC	LH	VE	BC	VC	VE	VC	VE	VB	VD	VC	VD	LH	VB	FD	FE	VE	VD	BE	VB	LH	LH	VC	LH	VB	LV	VE	VE	VC	LV	VE	LV	VC
VD	VD	VD	BB	LH	FB	BB	LV	LV	VD	FD	VD	VC	VD	LH	VE	LH	VB	VE	FC	LV	FC	BB	LV	LV	LH	VC	VE	BE	VC	FC	LH	FB	VD	VE	VC	VB	LH	LH	VB	VE	BE	LV	LV	BE
VB	LH	VB	FE	VC	VB	LH	FD	LV	BC	LH	VC	LH	VE	LV	VB	LV	VB	VB	LH	VD	LV	VB	FD	LV	BD	LH	VB	FB	VC	VD	VD	VE	BE	VD	VE	LH	VD	VD	VD	VC	LH	VB	VD	VC
LH	VD	LH	BE	FD	LV	BD	LH	VE	VE	FB	VE	BC	VE	VE	FE	BC	VD	LH	BC	VE	FC	VE	LV	VC	VC	LH	LV	VD	LH	VC	LV	VE	LH	LV	LV	VD	VD	VC	LV	VE	VE	VB	VB	VC
LH	LH	FD	VE	LV	VC	LH	VC	VD	VB	LV	LH	BE	VC	VB	VD	BB	LV	LH	FE	LH	VD	BB	FC	VE	LH	LV	VE	VB	VB	VB	LH	LH	BB	FB	LV	VD	VC	VC	LH	VE	LV	FB	LV	VE
LH	VB	BD	VB	VE	FE	VB	VC	LV	VD	VD	LH	VD	VD	VD	LV	FE	VD	LH	VE	VD	VC	BB	BD	LH	BB	FD	VD	VB	FE	VB	VB	FC	VE	VD	VE	VB	LV	LV	VB	VE	LH	VE	LV	VB
BC	VC	VB	VC	VB	LV	LH	LH	VD	LV	VD	VB	LH	FE	LV	VE	LV	BE	FC	LH	VB	VD	FD	VE	VD	FB	VB	BB	VB	VE	LH	BB	BC	VD	VC	LH	VE	VD	VB	LV	VB	VD	LV	FB	LV
LV	VD	BE	LV	LH	LH	LV	FB	LH	BD	FC	LV	LV	VB	BB	VD	FD	VC	LH	VC	VB	LH	LV	VD	BD	LH	VC	LV	VB	VC	VB	VE	VE	VB	VD	LH	LV	VB	VB	BD	VB	LH	BB	VD	LV
LV	FE	VD	VB	VE	VC	LH	LH	VB	VB	VB	LV	VC	LV	VD	VE	BB	LV	VC	LV	LV	LV	VE	LH	VE	LV	FB	VD	VC	VB	VD	LV	LH	VD	VC	VB	VC	VC	VB	VC	LH	VD	FD	VE	BD
LV	VD	LV	VE	LH	LV	FE	VC	LV	BC	BC	LV	FC	VC	LH	FD	LV	LV	LV	FD	BC	FE	VD	BE	LV	LH	LV	BB	VD	VC	VD	LV	FC	LV	FD	LV	VB	LV	LV	LV	VC	LV	LV	LH	FB
BD	VB	VE	FE	VB	VD	VD	BC	VD	LV	LH	VD	LV	BE	LV	VD	BC	LH	BC	LV	VB	VB	LV	VE	FB	LH	LV	BB	FD	VD	BC	BC	LV	VB	VC	VC	VE	LV	LV	VC	BE	BC	VC	VC	VE
LV	FE	BC	BC	LV	BC	LV	VE	FE	LH	VB	FD	VC	VD	VD	VB	FB	LV	LH	VC	LV	VD	FD	BB	LH	LV	VC	LV	VC	VB	FE	VC	LV	LV	LV	VB	LH	VB	FE	LV	FC	VB	LH	LV	BC
VE	VC	FE	BD	FE	VE	VE	VC	VE	BC	LH	VE	VC	VE	VB	VD	VC	VB	LV	VE	VD	VC	LH	LV	VE	FD	BE	FC	LV	VE	VB	BE	LV	VC	FB	VD	VB	BE	VE	VC	LV	VE	FE	VD	LH
FD	LH	VE	LH	LV	LH	VD	VE	VE	VD	FE	VB	LV	BE	VC	VB	LV	FC	LH	VE	BC	VE	VD	VC	VE	VC	VE	LV	VB	FD	LV	VB	VD	LV	BC	LV	LH	VB	VD	LH	LV	BB	LH	VE	LH
BB	VC	LV	VE	LV	LV	VC	LH	VB	LV	LH	LH	VB	VB	VE	LV	LV	LH	VE	BC	FE	VE	VE	VD	LV	VC	VE	LV	BC	VB	VC	VB	LH	LH	VC	VD	FD	BB	VC	FB	VB	BE	VC	VC	LH
VD	LV	VB	FE	VE	LH	VC	VC	VC	FB	BC	VB	VE	VD	FD	LV	LH	VD	VC	FB	LH	VD	VD	VB	FC	VD	VD	FE	VC	LV	LV	VD	LV	LH	FE	VD	BE	VE	VE	VB	LH	FD	LV	VD	VE
FB	VE	VC	BC	VD	VD	VC	VD	BE	BB	VC	VC	VD	VC	VE	LH	VD	LH	VD	VE	BB	VC	VD	BB	LV	VE	VB	BB	LH	LH	FB	VB	VE	LV	LV	VE	VB	VE	LH	VD	LH	VE	VC	VE	VB
VC	VE	LH	VE	VE	LH	LH	LH	LV	FE	VB	VE	LH	FD	LV	VC	BC	LV	VD	LV	FB	LH	VC	VE	VE	LH	BE	LV	LV	LV	LV	BE	FD	LV	LH	LV	VC	VC	FC	BB	LV	VE	VC	LV	LV
LH	VC	VB	FB	VD	VB	VC	VD	LH	VB	VC	LV	VB	VE	VD	LV	VB	FC	LH	LV	VC	VE	VE	VC	VB	VC	FE	VB	VC	VB	VE	VB	LH	VB	BB	VD	VD	LH	VE	VC	BD	FB	VC	VB	LV
BC	VD	VC	BE	LV	LH	LV	VD	VD	VE	VD	VB	VE	LH	BB	FC	VE	BD	VB	BB	LV	VE	FD	LH	FD	BE	LV	LH	VE	VE	VB	VD	VD	FE	LV	FC	VE	VC	VE	VE	VB	BB	VD	VC	VB
LV	VB	VD	FE	LH	VC	VC	VB	FC	VE	VE	LV	VD	FE	VE	VC	LV	VE	LH	VC	LH	BE	VD	VD	VC	VC	BD	LH	VD	LV	VE	FB	BE	LH	BB	VB	BE	LV	VB	VE	FC	BE	FD	LH	FC
LH	LH	VE	LV	VC	LH	VC	LH	VB	BD	VD	VC	BB	VB	VE	VB	LH	VD	VC	VD	VB	FC	VE	LH	BC	VB	FB	VB	BC	FE	VC	VD	VD	VE	VB	BC	FC	LV	BD	LV	LV	BE	VD	VC	LH
LH	LH	VB	VB	LH	BE	LV	VD	VB	VB	LH	LH	FE	VC	VB	VB	VD	LV	LH	VB	LV	LV	LH	LV	VB	LH	VE	VD	BD	BE	FE	VC	VE	LV	FC	LV	VD	VD	LV	VC	VD	FD	LH	VE	VE
VD	VB	VD	VD	VB	BC	FD	VB	VD	LH	VC	BD	VD	VE	VB	LV	VC	VC	VB	BB	LV	LH	LH	VC	FD	VE	VE	LH	LV	VC	LV	LV	VC	LH	LV	LH	LV	VD	VB	LV	VB	LH	VC	VE	VD
VD	LH	VE	VC	LV	VB	VD	LV	VC	VD	FC	LV	VE	VE	VE	VE	VE	VB	FD	VD	LH	VC	FC	VB	VD	FD	VD	LH	LH	LH	VE	LH	VE	LV	LV	LV	LV	VD	VB	LV	VC	LV	BC	LV	VC
BD	VC	FE	VB	VD	LV	VD	FD	LV	BB	VB	BB	FD	BC	FC	VC	VE	VE	BC	FE	VB	LV	VC	VE	VE	LV	BB	LH	FB	LH	LV	VD	LH	VB	BD	VC	VC	VC	FE	LH	LV	VE	VE	VB	VD
FC	VB	LV	VC	LV	FB	BB	VD	VE	VC	VD	VE	VC	VC	VE	VE	VD	LV	BE	LH	BC	LH	VC	BB	LV	FE	LH	BB	LH	VE	LV	VC	LH	VE	LV	VB	VB	LV	VD	VE	FB	VC	LV	VB	LH
VB	LH	LH	BC	VD	VD	LH	VD	VD	LV	VE	VE	LH	FB	VD	VB	LV	LH	LV	VE	LV	LH	LV	VD	LH	VB	LV	VE	BB	FB	VE	VB	VE	VE	FE	LH	LH	LV	LH	VE	LH	LH	LH	LV	BD
LH	FE	VD	LV	VE	VC	VB	VE	FB	LH	LV	LV	VB	BC	LH	BB	LH	VE	VB	VC	FD	LH	VD	BE	FC	LH	VE	BC	VD	VD	LH	VC	LH	VB	LH	VE	VD	VE	LH	LV	VD	VD	VD	VC	LH
BC	VB	VC	VD	LH	VE	LV	VC	BB	VC	LH	VC	VB	FD	VD	LV	VB	LV	LV	VB	LV	VB	VD	LH	LV	BC	FD	LV	VB	VE	FC	BD	VE	LH	VD	LV	VC	VC	VC	LV	LH	VD	VC	LV	LV
FD	VD	LH	LH	BB	VB	FE	LV	BD	LV	VE	FC	BE	LH	VD	FD	BD	VB	VD	VC	FE	LH	BC	LV	LV	LV	LV	VD	VE	LH	LV	VD	FE	BB	BC	FD	VE	BD	LV	VD	VE	LV	FC	VE	VB
