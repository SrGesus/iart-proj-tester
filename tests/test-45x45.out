VB	BB	LH	VE	VB	VE	VB	LH	VE	VB	VE	VB	LH	LH	BB	FE	VB	LH	LH	LH	LH	LH	LH	LH	BB	LH	LH	LH	LH	FE	VB	BB	LH	LH	LH	LH	VE	FD	LH	BB	VE	VB	LH	BB	VE
FC	LV	FD	BC	BE	FC	LV	FB	VD	VC	VD	VC	VB	VE	LV	VB	BC	LH	VE	FB	VB	LH	LH	LH	BE	VB	LH	LH	LH	LH	VC	VD	LH	LH	VE	VB	VC	VB	VE	FC	BD	VC	FB	LV	LV
VB	VC	VB	VE	FC	VB	VC	VD	BB	LH	VE	VB	VC	FC	BD	VC	VB	VE	VD	VC	LV	VB	FE	VB	VC	VD	LH	VE	VB	LH	BB	LH	FE	VB	VC	VD	LH	VC	VD	VE	VD	VE	LV	LV	LV
BD	VE	LV	VD	LH	VC	VB	VE	LV	FD	BC	VC	VB	VE	VD	FE	LV	LV	VB	LH	VC	BD	LH	VC	VB	LH	VE	VD	VC	VB	VC	VB	LH	VC	VB	BB	LH	FE	VB	VC	FB	VD	BE	LV	LV
FC	LV	LV	VB	VE	VB	BE	LV	LV	VB	LH	VE	LV	VD	LH	VE	LV	LV	LV	VB	FE	LV	VB	FE	VD	VE	LV	VB	VE	VD	VE	LV	VB	LH	VC	VD	BB	VE	LV	VB	BC	LH	VC	LV	LV
VB	VC	LV	LV	FC	LV	LV	VD	BE	LV	VB	VC	BD	VE	FD	BC	VC	VD	VC	BD	LH	VC	VD	LH	VE	LV	LV	LV	BD	FE	LV	LV	FC	VB	LH	VE	FC	LV	LV	FC	VB	LH	LH	VC	LV
LV	VB	VC	VD	LH	VC	LV	FB	LV	LV	VD	VE	FC	VD	LH	LH	VE	VB	VE	LV	FD	BB	BB	LH	VC	LV	LV	LV	VD	LH	VC	LV	VB	BC	FE	LV	VB	VC	VD	VE	LV	FD	VE	VB	VC
BD	VC	VB	BB	LH	VE	LV	LV	LV	LV	FB	VD	VE	VB	LH	LH	VC	FC	LV	VD	VE	FC	LV	VB	LH	VC	VD	BC	BB	FE	VB	VC	LV	VB	LH	VC	VD	BB	VE	LV	VD	VE	VD	BC	FE
VD	VE	FC	LV	FD	BE	LV	LV	LV	LV	VD	VE	LV	LV	VB	BB	LH	BB	VC	VB	BC	LH	VC	VD	VE	FB	VB	VE	FC	VB	BE	VB	VC	VD	LH	LH	VE	FC	VD	BC	VE	VD	LH	BB	VE
FB	VD	VE	BD	VE	VD	VC	VD	VC	VD	VE	LV	LV	LV	LV	VD	VE	FC	VB	VC	VB	LH	BB	FE	VD	BE	LV	VD	VE	LV	LV	VD	VE	VB	LH	FE	VD	LH	VE	VB	VC	VB	VE	FC	LV
BD	VE	LV	LV	VD	VE	VB	VE	VB	VE	LV	LV	LV	LV	VD	VE	VD	LH	BC	FE	LV	FB	VD	LH	VE	LV	VD	VE	LV	LV	VD	FE	LV	LV	VB	BB	LH	FE	LV	VD	VE	LV	LV	VB	BE
LV	VD	VC	VD	FE	LV	LV	VD	VC	VD	VC	LV	VD	VC	VB	VC	VB	LH	LH	LH	BC	BC	LH	VE	VD	VC	VB	VC	LV	VD	LH	VE	LV	VD	BE	VD	LH	VE	LV	FB	VD	VC	VD	VC	LV
LV	FD	LH	BB	LH	BE	VD	LH	VE	FD	LH	BC	LH	LH	BC	VE	LV	VB	VE	VB	LH	VE	VB	VC	VB	LH	VC	VB	BC	VE	VB	BE	VD	VE	FC	VB	LH	BE	LV	VD	VE	VB	LH	VE	LV
VD	LH	LH	VC	FB	VD	VE	FB	VD	LH	LH	LH	LH	VE	FB	LV	VD	VC	LV	LV	FD	BE	VD	VE	FC	VB	LH	VC	FD	VC	LV	VD	FE	VD	LH	BE	FD	VC	LV	VB	VC	LV	FB	VD	VC
FB	VB	LH	VE	BD	LH	VC	VD	LH	LH	BB	BB	FE	LV	VD	BC	VE	VB	VC	VD	VE	VD	VE	VD	LH	VC	FB	FD	VE	VB	BE	VB	LH	LH	VE	LV	VB	LH	VC	VD	VE	LV	VD	LH	VE
VD	VC	VB	BE	LV	FD	BB	LH	LH	VE	FC	VD	VE	VD	LH	VE	LV	VD	VE	FB	LV	FD	BC	LH	LH	LH	VC	VB	BC	VC	FC	LV	FD	VE	VD	VC	VD	LH	LH	VE	VD	BC	LH	LH	BE
VB	LH	VC	FC	VD	VE	LV	FD	LH	BC	LH	VE	LV	VB	LH	VC	LV	VB	VC	LV	VD	LH	VE	FD	LH	BB	LH	VC	FB	VB	VE	VD	VE	BD	VE	VB	LH	VE	VB	VC	VB	LH	VE	VB	VC
LV	VB	LH	BB	FE	LV	BD	LH	VE	VB	FE	VD	BE	VD	VE	FD	BE	VD	LH	BC	VE	FB	VD	LH	VE	VD	LH	LH	VC	LV	VD	LH	VC	LV	LV	LV	VB	VC	VD	LH	VC	VB	VC	VD	VE
LV	LV	FB	VD	LH	VC	LV	VB	VC	VD	LH	LH	BC	VE	VD	VE	BD	LH	LH	FE	LV	VD	BB	FE	VD	LH	LH	VE	VB	VC	VB	LH	LH	BE	FC	LV	VD	VE	VB	LH	VE	LV	FD	LH	VC
LV	VD	BE	VB	VE	FD	VC	VD	LH	VE	VB	LH	VE	VD	VE	LV	FC	VB	LH	VE	VD	VE	BD	BB	LH	BB	FE	VD	VC	FB	VD	VE	FB	VD	VE	VD	VE	LV	LV	VB	VC	LV	VB	LH	VE
BD	VE	VD	VC	VD	LH	LH	LH	VE	LV	VD	VE	LV	FB	LV	VD	LH	BC	FE	LV	VB	VC	FC	VD	VE	FC	VB	BB	VE	VD	LH	BC	BC	VE	VD	LH	VC	VD	VC	LV	VB	VC	LV	FB	LV
LV	VD	BB	LH	LH	LH	LH	FE	LV	BD	FE	LV	LV	VD	BC	VE	FB	VB	LH	VC	VD	LH	LH	VE	BD	LH	VC	LV	VD	VE	VB	VE	VB	VC	VB	LH	LH	VE	VB	BE	VD	LH	BC	VC	LV
LV	FD	VC	VB	VE	VB	LH	LH	VC	VD	VE	LV	VD	LH	VE	VD	BE	LV	VB	LH	LH	LH	VE	LV	VD	LH	FE	VD	VE	VD	VC	LV	LV	VB	VC	VB	VE	VD	VC	VD	LH	VE	FB	VB	BE
LV	VB	LH	VC	LV	LV	FB	VB	LH	BB	BE	LV	FD	VE	LV	FB	LV	LV	LV	FD	BB	FE	VD	BC	LH	LH	LH	BB	VC	VB	VE	LV	FC	LV	FB	LV	VD	LH	LH	LH	VE	LV	LV	LV	FC
BD	VC	VB	FE	VD	VC	VD	BC	VE	LV	LV	VD	LH	BE	LV	VD	BE	LV	BD	LH	VC	VB	LH	VE	FD	LH	LH	BE	FD	VC	BD	BC	LH	VC	VD	VC	VB	LH	LH	VE	BD	BC	VC	VD	VE
LV	FD	BC	BB	LH	BB	LH	VE	FC	LV	VD	FE	VB	VC	VD	VE	FC	LV	LV	VB	LH	VC	FD	BC	LH	LH	VE	LV	VB	VE	FC	VB	LH	LH	LH	VE	LV	VB	FE	LV	FC	VB	LH	LH	BE
VD	VE	FD	BC	FE	VD	VE	VD	VE	BD	LH	VE	VD	VE	VB	VC	VB	VC	LV	VD	VE	VB	LH	LH	VE	FD	BE	FC	LV	VD	VE	BD	LH	VE	FB	VD	VC	BD	VE	VD	LH	VC	FD	VE	LV
FB	LV	VB	LH	LH	LH	VC	VB	VC	VD	FE	VD	LH	BE	VD	VE	LV	FB	LV	VB	BC	VC	VB	VE	VD	VE	VD	LH	VC	FB	LV	VD	VE	LV	BD	LH	LH	VC	VD	LH	LH	BB	LH	VC	LV
BD	VC	LV	VB	LH	LH	VE	LV	VB	LH	LH	LH	VE	VD	VE	LV	LV	LV	VD	BE	FB	VB	VC	VD	LH	VC	VB	LH	BB	VC	VD	VE	LV	LV	VD	VE	FD	BB	VE	FB	VB	BE	VB	VE	LV
VD	LH	VC	FC	VB	LH	VC	VD	VC	FD	BB	VE	VD	VE	FC	LV	LV	VD	VE	FC	LV	VD	VE	VB	FE	VB	VC	FB	VD	LH	LH	VC	LV	LV	FB	VD	BB	VC	VD	VC	LV	FC	LV	VD	VC
FD	VE	VB	BB	VC	VB	VE	VB	BB	BB	VC	VD	VE	VD	VE	LV	VD	LH	VC	VB	BE	VB	VC	BD	LH	VC	VB	BC	LH	LH	FE	VB	VC	LV	LV	VB	VC	VB	LH	VE	LV	VB	VC	VB	VE
VB	VC	LV	VD	VE	LV	LV	LV	LV	FC	VB	VE	LV	FB	LV	VD	BB	LH	VE	LV	FC	LV	VB	VC	VB	LH	BC	LH	LH	LH	LH	BC	FE	LV	LV	LV	VB	VC	FD	BE	LV	VD	VE	LV	LV
LV	VB	VC	FB	VD	VC	VD	VC	LV	VB	VC	LV	VD	VC	VD	LH	VC	FB	LV	LV	VB	VC	VD	VE	VD	VE	FB	VB	VE	VB	VE	VB	LH	VC	BD	VC	VD	LH	VE	VD	BE	FB	VD	VC	LV
BD	VC	VB	BC	LH	LH	LH	VE	VD	VC	VB	VC	VB	LH	BB	FE	VB	BE	VD	BE	LV	VB	FE	LV	FD	BE	LV	LV	VD	VC	VD	VC	VB	FE	LV	FD	VE	VB	VC	VB	VC	BD	VE	VB	VC
LV	VB	VC	FD	LH	VE	VB	VC	FD	VE	VD	LH	VC	FB	VD	VE	LV	VD	LH	VC	LV	BD	VE	VD	VE	VD	BE	LV	VB	LH	VE	FD	BC	LH	BE	VB	BE	LV	VB	VC	FD	BE	FC	LV	FB
LV	LV	VB	LH	VE	LV	VD	LH	VE	BD	VE	VB	BB	VC	VB	VC	LV	VB	VE	VB	VC	FC	VD	LH	BC	VE	FC	VD	BE	FB	VD	VE	VB	VE	VD	BE	FC	LV	BD	LH	LH	BE	VB	VC	LV
LV	LV	VD	VE	LV	BD	LH	VE	VD	VC	LV	LV	FC	VB	VC	VB	VC	LV	LV	VD	LH	LH	LH	LH	VE	LV	VB	VE	BD	BC	FE	VD	VC	LV	FB	LV	VB	VC	LV	VB	VE	FC	LV	VB	VC
VD	VC	VB	VC	VD	BE	FD	VC	VB	LH	VC	BD	VE	VD	VE	LV	VB	VC	VD	BB	LH	LH	LH	VE	FC	VD	VC	LV	LV	VB	LH	LH	VE	LV	LV	LV	LV	VB	VC	LV	VD	LH	VC	VD	VE
VB	LH	VC	VB	LH	VC	VB	LH	VC	VB	FE	LV	VD	VE	VD	VC	VD	VE	FB	VD	LH	VE	FB	VD	VE	FD	VE	LV	LV	LV	VB	LH	VC	LV	LV	LV	LV	VD	VE	LV	VB	LH	BB	LH	VC
BD	VE	FD	VC	VB	LH	VC	FD	LH	BC	VE	BD	FE	BD	FE	VB	VE	VD	BE	FD	VE	LV	VD	VE	VD	LH	BE	LV	FC	LV	LV	VB	LH	VC	BD	VC	VD	VE	FC	LV	LV	VB	VC	VB	VE
FC	VD	LH	VE	LV	FD	BB	VE	VB	VE	VD	VC	VB	VC	VB	VC	VD	LH	BC	LH	BE	LV	VB	BC	LH	FE	LV	BD	LH	VC	LV	VD	LH	VE	LV	VB	VE	LV	VB	VC	FC	VD	LH	VC	LV
VB	LH	LH	BE	VD	VE	LV	VD	VC	LV	VB	VE	LV	FD	VC	VB	LH	LH	LH	VE	LV	LV	LV	VB	LH	VE	LV	VD	BB	FE	VD	VE	VB	VC	FC	LV	LV	LV	LV	VB	LH	LH	LH	LH	BE
LV	FD	VE	LV	VB	VC	VD	VE	FB	LV	LV	LV	VD	BB	LH	BC	LH	VE	VB	VC	FC	LV	VD	BC	FE	LV	VD	BB	VC	VB	LH	VC	LV	VB	LH	VC	VD	VC	LV	LV	VB	VE	VB	VE	LV
BD	VE	VD	VC	LV	VB	LH	VC	BD	VC	LV	VD	VE	FC	VB	LH	VE	LV	LV	VB	LH	VC	VB	LH	LH	BC	FE	LV	VB	VC	FD	BB	VC	LV	VB	LH	VE	VB	VC	LV	LV	VD	VC	LV	LV
FC	VD	LH	LH	BC	VC	FD	LH	BC	LH	VC	FD	BC	LH	VC	FD	BC	VC	VD	VC	FD	LH	BC	LH	LH	LH	LH	VC	VD	LH	LH	VC	FD	BC	BC	FE	VD	BC	LH	VC	VD	LH	FE	VD	VC
